diff --git a/dist/game/data/scripts/quests/Q00255_Tutorial/Q00255_Tutorial.java b/dist/game/data/scripts/quests/Q00255_Tutorial/Q00255_Tutorial.java
index e16093d456f..c7b69956330 100644
--- a/dist/game/data/scripts/quests/Q00255_Tutorial/Q00255_Tutorial.java
+++ b/dist/game/data/scripts/quests/Q00255_Tutorial/Q00255_Tutorial.java
@@ -3564,6 +3564,11 @@ public class Q00255_Tutorial extends Quest
 	public void onKill(Npc npc, Player killer, boolean isSummon)
 	{
 		final QuestState qs = getQuestState(killer, false);
+		if (qs == null)
+		{
+			return;
+		}
+
 		if (npc.getId() == TUTORIAL_GREMLIN)
 		{
 			if ((qs.getMemoStateEx(1) == 1) || (qs.getMemoStateEx(1) == 0))
